src/containers/Dashboard/index.js

```diff
@@ -15,8 +15,16 @@ class Dashboard extends Component {
       updatedTimeRange,
       fetchDashboardData
     } = this.props;
-    updatedTimeRange(range);
-    fetchDashboardData();
+
+    // data live reloads every 5 seconds
+    setInterval(
+      (function fetchData() {
```

Comment:
Isn't it same as:
```
setInternal(
  function fetchData() {
    updatedTimeRange(range);
    fetchDashboardData();
  },
  5000
);
```

Reply:
I made it a self-invoking function because I want `fetchData` to run immediately and then be called every 5 seconds.